gammapy/data/tests/test_data_store.py

```diff
@@ -91,22 +95,63 @@ def test_datastore_subset(tmpdir, data_manager):
 
     assert str(actual.events) == str(desired.events)
 
-    #Copy only certain HDU classes
+    # Copy only certain HDU classes
     storedir = tmpdir / 'substore2'
-    data_store.copy_obs(obs_id, storedir, hdu_class = ['events'])
-    
+    data_store.copy_obs(obs_id, storedir, hdu_class=['events'])
+
     substore = DataStore.from_dir(storedir)
     assert len(substore.hdu_table) == 2
 
+
 @requires_data('gammapy-extra')
 @requires_dependency('yaml')
 def test_data_summary(data_manager):
     """Test data summary function"""
-    
+
     data_store = data_manager['hess-crab4-hd-hap-prod2']
     t = data_store.data_summary([23523, 23592])
     assert t[0]['events'] == 620975
     assert t[1]['edisp_2d'] == 28931
 
     t = data_store.data_summary([23523, 23592], summed=True)
     assert t[0]['psf_3gauss'] == 6042
+
+
+@pytest.mark.parametrize("pars,result", [(dict(energy=None, theta=None),
```

Comment:
Yes, this is exactly what I had in mind.
Thank you!

Can you try to make the content in `parametrize` a little more readable (i.e. not so far on the right side)?
Maybe just adding a newline after the opening `[` would be helpful, i.e. something like this:
```
@pytest.mark.parametrize("pars,result", [
    "put stuff here"
])
```